src/main/resources/static/openapi/SupportAdditionalNeedsAPI.yml

```diff
@@ -1473,6 +1492,25 @@ components:
           description: |
             An ISO-8601 date representing date that the plan creation is due.
           example: '2023-11-19'
+        planCompletedDate:
+          type: string
+          format: date
+          description: |
+            An ISO-8601 date representing date that the plan creation was created.
+          example: '2023-11-19'
+        planCompletedBy:
+          type: string
+          description: |
+            The name of the person who created the plan, this will be blank 
+            if the person who keyed in the plan is the same person doing the review.
+        planKeyedInBy:
+          type: string
+          description: |
+            The name of the person who created tha pan in SAN.
```

Comment:
How about

> If the status of this Plan Creation Schedule is COMPLETED, this field is the DPS username of the user that keyed the Education Support Plan into the system.
This field will only have a value when the status of the Plan Creation Schedule is COMPLETED, and reflects the logged in user who interacted with the SAN service.

Reply:
```
example: AUSER_GEN
```